app/views/councillor_contributions/_contribution_form_input.html.haml

```diff
@@ -7,3 +7,4 @@
       .suggested-councillor-input-group
         = suggested_councillor_field.label :email, class: "councillor-contribution-label"
         = suggested_councillor_field.text_field :email
+      = @councillor_contribution.suggested_councillors.last.errors.full_messages.to_sentence
```

Comment:
> This error display is pretty basic and it's not currently prominent and obvious to the contributor that something's gone wrong. At a minimum I think we need to move it somewhere more prominent and semantically appropriate.

One idea might be to have:

1. message at the top of the form like "There's a problem with the information you entered. See the messages below and resolve the issue before submitting your councillors." 
2. "We could also style it with form error styling we use elsewhere in the application." that plus the detailed input error message below the input.

Here's some stuff on accessible form error message html https://www.paciellogroup.com/blog/2016/01/simple-inline-error-message-pattern/ :)

Here's an example of how mailchimp do something like this, I think we can be more helpful and friendly :):

![screen shot 2017-06-09 at 9 43 14 am](https://user-images.githubusercontent.com/1239550/28955171-d12a2b4a-7927-11e7-8dfd-ce41cb45f21f.png)